mne/time_frequency/csd.py

```diff
@@ -1305,7 +1308,7 @@ def _compute_mt_params(n_times, sfreq, bandwidth, low_bias, adaptive):
     if bandwidth is not None:
         half_nbw = float(bandwidth) * n_times / (2. * sfreq)
     else:
-        half_nbw = 2.
+        half_nbw = 4.
```

Comment:
@larsoner can you explain this change?

Reply:
It was necessary to make PSD/CSD equal using defaults. It is `4.` everywhere else in our code, except maybe one place in `minimum_norm/time_frequency.py`.

Reply:
I have the feeling that _compute_mt_params should have been the multitaper.py file so it can be common between functions that use multitapers....

can you have a look?

@wmvanvliet ok for you about this change?

Reply:
Yes I can try to unify these

Reply:
ok for me!